card.go

```diff
@@ -476,6 +476,13 @@ func getCachedOrStoreCard(card *Card, ncn string) (Card, error) {
 func getScryfallCard(input string) (Card, error) {
 	cardRequests.Add(1)
 	var card Card
+	
+	// See if it's a known short name of a card.
+	if qualifiedName, ok := shortCardNames[input]; ok {
```

Comment:
Since we're not normalising here does that mean "tnn" won't match "TNN" etc?

Reply:
Fixed. Moved normalisation above, verified all names in short_names.json are lowercase.